pageTests/api/book-a-visit.test.js

```diff
@@ -13,6 +13,10 @@ describe("schedule-visit", () => {
     ward: "MEOW",
   }));
 
+  const getUpdateWardVisitTotalsSpy = jest.fn(() => {
```

Comment:
This would be an `updateWardVisitTotalsSpy` right?

Reply:
yes :eyes: